### PR TITLE
Continuous integration on Windows (appveyor)

### PR DESCRIPTION
**Pull request by alubbock**
This PR adds continuous integration on Windows using the [Appveyor](https://ci.appveyor.com/) platform. Enable using the Appveyor Github integration.

Notes:

- I've set up tests for Python 2.7 and Python 3.4 on win32 platform
- PyGraphviz binaries from [Chris Gohlke](http://www.lfd.uci.edu/~gohlke/pythonlibs/) are used, but can't be downloaded directly, so I used Dropbox. The files are <1Mb so performance shouldn't be impacted.
- Kappa doesn't have a `KaSa` binary available for Windows AFAIK, so the Kappa static analysis tests are skipped on that platform (Kappa simulations are still tested)
- Anaconda doesn't have a numpy 1.12 binary for Python 3.4/win32. I added doctest skip declarations to two tests in simulator/base.by to make the test suite pass on numpy 1.11.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - activate test-environment
+
+  # Majority of dependencies can be installed with Anaconda
+  - conda install numpy scipy matplotlib sympy networkx nose h5py pandas
+
+  # Graphviz & PyGraphviz
+  - appveyor DownloadFile "http://graphviz.org/pub/graphviz/stable/windows/graphviz-2.38.zip" -FileName graphviz.zip
+  - 7z x graphviz.zip -y > nul
+  - set "PATH=%cd%\release\lib\release\dll;%PATH%"
+  - pip install https://dl.dropboxusercontent.com/u/6251352/pygraphviz-1.3.1-cp%PYTHON_SVER%-none-win32.whl?dl=1
```
Comment (jmuhlich) [inline appveyor.yml]: I found a conda channel offering pygraphviz 1.3.1 for win32/win64: `marufr`. So `conda install -c marufr pygraphviz` ought to do it, then we won't need to maintain this in dropbox.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 2.7
+      PYTHON_SVER: 27
+      MINICONDA: C:\Miniconda
+    - PYTHON_VERSION: 3.4
```
Comment (jmuhlich) [inline appveyor.yml]: Why not test on 3.5 or 3.6 (or both)? Appveyor does offer both:
https://www.appveyor.com/docs/build-environment/#miniconda
Then we could have numpy 1.12 and avoid dropping those doctests.

Comment (alubbock) [inline appveyor.yml]: The limiting factor is the pygraphviz binary - I haven't been able to find one for 3.5 or 3.6 on win32, so I had to go with Chris Gohlke's 3.4 build.

File: appveyor.yml (hunk 1 above)
Comment (alubbock) [inline appveyor.yml]: `marufr`'s build is Python 2.7 only, unfortunately.

**Comment by alubbock** [comment]
PySB recently broke compatibility with Kappa 3.5, so that'll need updating to 4 in Appveyor. Need to wait for the Kappa team to release a 4 binary for Windows. There's a pending issue for this: Kappa-Dev/KaSim#394.

File: appveyor.yml (hunk 2 above)
Comment (jmuhlich) [inline appveyor.yml]: ARRGH!  :)

**Comment by jmuhlich** [comment]
OK I guess we will have to let this sit until Kappa releases the v4 windows binary.

**Pull request merged**